Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -819,7 +819,7 @@ public extension _ChatChannelController {
             channelModificationFailed(completion)
             return
         }
-        guard cooldownDuration >= 1 && cooldownDuration <= 120 else {
+        guard cooldownDuration >= 1, cooldownDuration <= 120 else {
```

Comment:
Are we sure we want this configuration? i.e. using `,` instead of good old `&&` for conditions? imho in this particular case `&&` makes sense... What about opting-out this rule maybe?

Reply:
Done ✅  Disabled the rule 👍